nautobot_golden_config/nornir_plays/config_deployment.py

```diff
@@ -25,24 +27,24 @@
 
 
 @close_threaded_db_connections
-def run_deployment(task: Task, logger: logging.Logger, config_plan_qs, deploy_job_result) -> Result:
+def run_deployment(task: Task, logger: logging.Logger, config_plan_qs, deploy_job_result, job_request) -> Result:
```

Comment:
This changes the signature, which I think is okay, but just want to ensure we are purposeful and in agreement.

Reply:
hmm, looking at this closer. Wasn't intended. Thanks for finding

Reply:
Was purposeful, since I need that data to do post processing.

Reply:
Sorry, I mean, is it purposeful with consideration for a potential breaking function signature. We could use a kwarg as an example. I don't know that we need to consider that, but just making sure it is thought of before.

Reply:
I did go through it when I changed the signature and really only found the one place its getting called.

Reply:
okay, and anyone else calling this would be accessing code that is outside the documented API?

Reply:
That's my thought, I guess a potential option would be to do `**kwargs` instead? Which I think would allow for this to work and others using it in an undocumented fashion.